src/Files.App/UserControls/StatusBarControl.xaml

```diff
@@ -163,26 +163,47 @@
 			Orientation="Horizontal"
 			Spacing="4">
 
-			<!--  Open in VS Code Button  -->
+			<!--  Open in IDE Button  -->
 			<Button
-				x:Name="OpenInVSCodeButton"
+				x:Name="OpenInIDEButton"
 				Height="24"
 				Padding="8,0,8,0"
 				VerticalAlignment="Center"
-				x:Load="{x:Bind converters:MultiBooleanConverter.AndNotConvert(Commands.OpenInVSCode.IsExecutable, Commands.OpenInVS.IsExecutable), Mode=OneWay}"
+				x:Load="{x:Bind Commands.OpenInVSCode.IsExecutable, Mode=OneWay}"
 				Background="Transparent"
-				BorderBrush="Transparent"
-				Command="{x:Bind Commands.OpenInVSCode}"
-				ToolTipService.ToolTip="{x:Bind Commands.OpenInVSCode.LabelWithHotKey, Mode=OneWay}">
+				BorderBrush="Transparent">
 				<Button.Content>
 					<StackPanel Orientation="Horizontal" Spacing="8">
 						<usercontrols:OpacityIcon
 							Width="16"
 							Height="16"
 							Style="{StaticResource ColorIconOpen}" />
-						<TextBlock Text="{x:Bind Commands.OpenInVSCode.Label, Mode=OneWay}" />
+						<TextBlock Text="{helpers:ResourceString Name=Open}" />
 					</StackPanel>
 				</Button.Content>
+
+				<Button.Flyout>
+					<MenuFlyout>
+						<MenuFlyoutItem
+							x:Name="OpenInVSButton"
+							x:Load="{x:Bind Commands.OpenInVS.IsExecutable, Mode=OneWay}"
```

Comment:
This doesn't seem to be working properly

Reply:
Really? I'll check again

Reply:
I fixed the issue, but on second thought I think we should remove the option. I'd rather let users choose the IDE instead of this.